public/app/ui/components/collections/table-results.html

```diff
@@ -0,0 +1,26 @@
+<table class="table table-bordered table-striped results-table">
```

Comment:
And again just for consistency / cleanliness I pulled the table view out into its own component.  It's still generic to all collections, just moved out of `ui/pages/search/index.html` into its own file.